download/download.go

```diff
@@ -83,7 +83,7 @@ func download(path, dest string, firstAttempt bool) (root *vcs.RepoRoot, err err
 	if err != nil && firstAttempt {
 		// may have been rebased; we delete the directory, then try one more time:
 		log.Printf("Failed to update %q (%v), trying again...", root.Repo, err.Error())
-		err = os.RemoveAll(fullLocalPath)
+		os.RemoveAll(fullLocalPath)
```

Comment:
Let's log the `err` here rather than remove it

Reply:
Sure. Can do that